src/docs/product/releases/setup/release-automation/jenkins/index.mdx

```diff
@@ -63,7 +63,7 @@ pipeline {
             }
             steps {
                 // Install Sentry CLI
-                sh 'curl -sL https://sentry.io/get-cli/ | bash'
+                sh 'curl -sL https://sentry.io/get-cli/ | bash || true'
```

Comment:
Is this more what you're trying to say? If sentry-cli isn't installed, then we should install it. This won't fail if sentry-cli isn't installed, assuming the installation succeeds.

```suggestion
                sh 'command -v sentry-cli || curl -sL https://sentry.io/get-cli/ | bash'
```